integration-tests/tests/run-analysis-azure.ts

```diff
@@ -79,6 +79,9 @@ const testRunAnalysisAzure = _.flowRight(
   const frame = await findIframe(page, '//iframe[@title="Interactive JupyterLab iframe"]', {
     timeout: Millis.ofMinutes(2),
   });
+  if (!frame) {
```

Comment:
If it doesn't find the frame, doesn't it throw an exception already?

Reply:
It doesn't seem to - https://app.circleci.com/pipelines/github/DataBiosphere/terra-ui/21178/workflows/70974172-0d22-484a-92bc-85a88489fca7/jobs/74532/tests 

the helper which gets the frame does a `find()` across the page's frames, not a waitForXPath, so I think a falsy result won't throw.